starfyre/__main__.py

```diff
@@ -42,16 +42,26 @@ def create_main_file(path):
 
 @click.command()
 @click.option("--path", default=".", help="Path to the project")
-@click.option("--dev", default=False, help="Start the compilation and generate the build package.")
-@click.option("--build", default=False, help="Start the build package")
+@click.option("--dev/--no-dev", default=False, help="Start the compilation and generate the build package.")
+@click.option("--build/--no-build", default=False, help="Start the build package")
 def main(path, dev, build):
+    """
+    Command-line interface to compile and build a project.
+
+    Args:
+        path (str): Path to the project directory.
+        dev (bool): Whether to start the compilation and generate the build package.
+        build (bool): Whether to start the build package.
```

Comment:
Can we have a `\n` at the end of lines here? As this docstring is visible when you type the command `python -m starfyre --help`.

So, e.g. 
```
path (str): Path to the project directory.\n
dev (bool): Whether to start the compilation and generate the build package.\n
```